tests/checkers/unittest_unicode/functions.py

```diff
@@ -0,0 +1,253 @@
+import itertools
```

Comment:
Shouldn't this file be named ``test_functions.py`` ?

Reply:
renamed